src/Template/Bake/Controller/controller.ctp

```diff
@@ -22,6 +23,22 @@ foreach ($components as $component):
 class <%= $name %>Controller extends AppController
 {
 <%
+echo $this->Bake->arrayProperty('helpers', $helpers, ['indent' => false]);
 echo $this->Bake->arrayProperty('components', $components, ['indent' => false]);
 %>
+<% if (!empty($actions)) : %>
+    public function beforeFilter(Event $event) {
+        parent::beforeFilter($event);
+
+        // you can remove all these mapActions if your AppController has mapped the actions via configuration
+        // read http://crud.readthedocs.org/en/latest/configuration.html#actions for details
+        // e.g. of a on-the-fly enable action
+        // $this->Crud->mapAction('add', 'Crud.Add');
+    }
+<% endif; %>
+<%
+foreach($actions as $action) {
+    echo $this->element('Controller/' . $action);
```

Comment:
If this template does not exist, I believe that the bake shell will default to use the cakephp/bake add template.

I have tested by deliberately not having a Bake folder inside FriendsOfCake/crud/src/Template and then still run `bin/cake bake controller users_in_apps --theme Crud`

I got a Error: "Crud" is not a valid value for --theme. Please use one of "Bake, Migrations"

I have tested by deliberately not having a `add.ctp` inside FriendsOfCake/crud/src/Template/Bake/Element/ and then still run `bin/cake bake controller users_in_apps --theme Crud`

the add action baked was that of the original one we expect from cakephp/bake.

Does that help to answer your question, @josegonzalez ?